### DISCUSSION TITLE
Model Repository

### DISCUSSION THREAD

**Discussion by Drake-AI** [LostRuins/koboldcpp#87] [Show and tell]
Models for use with koboldcpp

### CHAT MODELS

**Pygmalion 6B:** - https://huggingface.co/concedo/pygmalion-6bv3-ggml-ggjt

### ADVENTURE MODELS

**Skein 6B:** - https://huggingface.co/Kastor/GPT-J-6b-Skein-ggml-q4_1

**Adventure 6B:** - https://huggingface.co/Kastor/GPT-J-6B-Adventure-ggml-q4_1

**Adventien 6B (Adventure+Skein):** - https://huggingface.co/Kastor/GPT-J-6B-Adventien-ggml-q4_1

### NOVEL MODELS

**Janeway 6B:** - https://huggingface.co/concedo/janeway-6b-ggml

**Shinen 6B:** - https://huggingface.co/xzuyn/GPT-J-6B-Shinen-ggml-q4_0

### MIXED MODELS

**Pygway 6B:** - https://huggingface.co/Kastor/GPT-J-6B-Pygway-ggml-q4_1

**Javalion R:** - https://huggingface.co/xzuyn/Javalion-R-ggml-q4_0

**Alpacino 13B:** - https://huggingface.co/verymuchawful/Alpacino-13b-ggml

### GENERIC MODELS

**WizardLM 7B:** - https://huggingface.co/TheBloke/wizardLM-7B-GGML

(Posted by LostRuins)
**Cerebras 111M (GPT2):** - https://huggingface.co/concedo/cerebras-111M-ggml/

**RWKV-v4-169M:** - https://huggingface.co/concedo/rwkv-v4-169m-ggml/tree/main

(Posted by gustrd)
**Alpaca 7B:** - https://huggingface.co/Sosaka/Alpaca-native-4bit-ggml

**Alpaca 13B:** - https://huggingface.co/Pi3141/alpaca-native-13B-ggml

**GPT-X-Alpaca 13B:** - https://huggingface.co/anon8231489123/gpt4-x-alpaca-13b-native-4bit-128g

**Alpaca 30B:** - https://huggingface.co/Pi3141/alpaca-lora-30B-ggml

(Posted by noprotocolunit)
**Llama 13B:** - https://huggingface.co/camelids/llama-13b-ggml-q4_0

(Posted by Myrminki)
**gpt4all:** - https://huggingface.co/eachadea/ggml-gpt4all-7b-4bit

**Comment by LostRuins** [comment]
Cerebras 111M (GPT2) - https://huggingface.co/concedo/cerebras-111M-ggml/

RWKV-v4-169M - https://huggingface.co/concedo/rwkv-v4-169m-ggml/tree/main

**Comment by gustrd** [comment]
**Alpaca 7B:** https://huggingface.co/Sosaka/Alpaca-native-4bit-ggml

**Alpaca 13B:** https://huggingface.co/Pi3141/alpaca-native-13B-ggml

**GPT-X-Alpaca 13B:** https://huggingface.co/anon8231489123/gpt4-x-alpaca-13b-native-4bit-128g 

**Alpaca 30B:** https://huggingface.co/Pi3141/alpaca-lora-30B-ggml

I have tested then. Some must run the convert script to maximum performance.

**Comment by Drake-AI** [comment]
I will edit the first post to keep all models organized by categories. Thanks for the contributions.

**Comment by noprotocolunit** [comment]
This one works well (worked as of a few days ago): https://huggingface.co/camelids/llama-13b-ggml-q4_0 
It's just generic llama.

**Comment by Drake-AI** [comment]
Added 3 more models converted and tested by me, Adventure, Adventien and Pygway.

**Comment by Myrminki** [comment]
gpt4all: https://huggingface.co/eachadea/ggml-gpt4all-7b-4bit

**Comment by Drake-AI** [comment]
Added wizardLM 7B, great model, better than alpaca, give it a try.

**Comment by Myrminki** [comment]
Baize_lora_13b_q4: https://huggingface.co/couchpotato888/baize_lora_q4_ggml
